### DISCUSSION TITLE
tutorial06 suggestion

### DISCUSSION THREAD

**Discussion by ahmetakman** [lava-nc/lava#213] [Ideas]
Hello, 
I have been busy with the lava tutorial for some days, and I would like to add that they are very helpful to understand what is going on with this library. However, I think the block diagram illustrations should be available more. For example, if the sub-process model built at tutorial06 were illustrated it would be better for newcomers to have intuition about the structural architecture.

All the best,

**Comment by awintel** [comment]
Thanks for the feedback. 
For feedback/reports/... on existing code it's best to file an issue in the respective repo.

**Reply by ahmetakman** [reply]
Thanks for the reply, at some points i think "hmm that is not much of an issue but i should let them know". That is why I have hesitated for some discussions. Also the I have not get used to the computer I am using the lava libraries on (to avoid in case of redundant issues stemmed from local problems) ; so,  this is also a factor. I will be more careful about on this issue, sorry for the inconvinience.
